core-java-modules/core-java-9-jigsaw/src/test/java/com/baeldung/java9/modules/ModuleAPIUnitTest.java

```diff
@@ -2,8 +2,7 @@
 
```

Comment:
I have removed the assertions on the number of modules. It's dependent on the JVM version, so we shouldn't test for it.